remoteworkitem/trackeritem_repository.go

```diff
@@ -8,3 +8,18 @@ func upload(db *gorm.DB, tqID int, item map[string]string) error {
 	err := db.Create(&ti).Error
 	return err
 }
+
+func convert(db *gorm.DB, tqID int, item map[string]string, provider string) error {
```

Comment:
@baijum If this isn't put inside a transaction, then we could be calling this inside after upload(...) 
https://github.com/almighty/almighty-core/blob/master/remoteworkitem/scheduler.go#L46